### PR TITLE
CI: remove extra quotes

### PR DESCRIPTION
**Pull request by Jaakko-Os**
https://github.com/metal3-io/project-infra/pull/139#discussion_r479900953

### CODE REVIEW

**Comment by Jaakko-Os** [comment]
@maelk

File: jenkins/scripts/files/run_fetch_logs.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ do
 done
 
 mkdir -p "${LOGS_DIR}/qemu"
-sudo sh -c "cp -r /var/log/libvirt/qemu/* "${LOGS_DIR}/qemu/""
+sudo sh -c "cp -r /var/log/libvirt/qemu/* \"${LOGS_DIR}/qemu/\""
```
Comment (maelk) [inline jenkins/scripts/files/run_fetch_logs.sh:51]: you could also remove the inner quotes. they were mostly for the variable, and that's now taken care by the outer quotes.

Comment (Jaakko-Os) [reply jenkins/scripts/files/run_fetch_logs.sh]: done

**Comment by maelk** [comment]
Please rename the PR
/approve

**Pull request merged**